packages/usa-prose/src/styles/_usa-prose.scss

```diff
@@ -8,5 +8,20 @@
     @include usa-headings-styles;
     @include usa-list-styles;
     @include usa-table-styles;
+    
+    * + {
+      h1,
+      h2,
+      h3,
+      h4,
+      h5,
+      h6 {
+        margin-top: 1.5em;
```

Comment:
What do you think about adding a couple system variables to `settings-typography` that we can use whenever we want to reference these values for margin-top? It might not be necessary since there are only a couple references at the moment, but it will make it easier to be more consistent across files and would give some flexibility to the users. 

```sass
$theme-heading-margin-top: 1.5em !default;
$theme-paragraph-margin-top: 1em !default;
```